src/constants/card_constants.asm

```diff
@@ -10,384 +10,384 @@
 	const FULLHEAL_ENERGY         ; $009
 	const RAINBOW_ENERGY          ; $00a
 	const RECYCLE_ENERGY          ; $00b
-	const BULBASAUR1              ; $00c
-	const BULBASAUR2              ; $00d
-	const BULBASAUR3              ; $00e
-	const IVYSAUR1                ; $00f
-	const IVYSAUR2                ; $010
+	const BULBASAUR_LV12          ; $00c
+	const BULBASAUR_LV13          ; $00d
+	const BULBASAUR_LV15          ; $00e
+	const IVYSAUR_LV20            ; $00f
+	const IVYSAUR_LV26            ; $010
 	const DARK_IVYSAUR            ; $011
-	const VENUSAUR1               ; $012
-	const VENUSAUR2               ; $013
-	const VENUSAUR3               ; $014
+	const VENUSAUR_LV64           ; $012
+	const VENUSAUR_LV67           ; $013
+	const VENUSAUR_LV67A          ; $014
```

Comment:
I'm warming up to the idea of adding level to the constants but there may be a better suffix to distinguish pairs like these.

If we decide to adopt this change, then the card constants in poketcg 1 should be updated to match. Would you be able to open a similar PR to that repo to keep them in sync? (if not, one of us would get around to it)

Reply:
I have a list of the 62 names with numbers at the end, how would you like to handle referencing them?